pkg/controller/egress_router/egress_router_controller.go

```diff
@@ -89,6 +91,7 @@ func newEgressRouterReconciler(mgr manager.Manager, status *statusmanager.Status
 
 func (r EgressRouterReconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	klog.Infof("Reconciling egressrouter.network.operator.openshift.io %s\n", request.NamespacedName)
+	ctx = context.WithValue(ctx, names.KEY_CONTROLLER_NAME, "egressrouter")
```

Comment:
Making it a value on the Context theoretically improves debug output (since it's part of the context's String() output), but I'm not sure how useful that really is.

I'll add it as an explicit parameter.

Reply:
so you no longer need this `context.WithValue` call (or `names.KEY_CONTROLLER_NAME`)

Reply:
agreed, will remove.